src/gproc_dist.erl

```diff
@@ -79,7 +79,13 @@ start_link() ->
     start_link({[node()|nodes()], []}).
 
 start_link(all) ->
-    start_link({[node()|nodes()], [{bcast_type, all}]});
+    Workers = case application:get_env(gproc_dist_workers) of
```

Comment:
Why not just
```erlang
Workers = case application:get_env(gproc_dist_workers) of
    {ok, [_|_] = WorkersList} -> WorkersList;
    _ -> []
end
```
?

Reply:
Err. No good reason.

Is it otherwise okay?

Reply:
I believe so.

Reply:
I incorporated your change. Thanks!